cita-secp256k1/Cargo.toml

```diff
@@ -6,14 +6,14 @@ license = "Apache-2.0"
 edition = "2018"
 
 [dependencies]
-secp256k1 = {version = "0.12", features = ["rand"]}
+secp256k1 = {version = "0.15", features = ["rand", "recovery"]}
```

Comment:
Is the new version functionally compatible with the old version?

Reply:
You know I'm not sure. I think "almost"...

Reply:
Test passed , So it's ok.